routers/api/v1/api.go

```diff
@@ -890,6 +891,14 @@ func Routes() *web.Router {
 		// Misc (public accessible)
 		m.Group("", func() {
 			m.Get("/version", misc.Version)
+			m.Group("/explore", func() {
```

Comment:
Shouldn't the API route be /search/code to align with the naming conventions of the [GitHub API](https://docs.github.com/en/rest/search/search?apiVersion=2022-11-28#search-code)?

```suggestion
			m.Group("/search", func() {
```